src/main/java/uk/gov/justice/hmpps/probationteams/dto/LocalDeliveryUnit2Dto.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.hmpps.probationteams.dto
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+import javax.validation.constraints.Email
+import javax.validation.constraints.NotBlank
+
+@ApiModel(description = "Local Delivery Unit")
+data class LocalDeliveryUnit2Dto @JsonCreator constructor(
```

Comment:
Not part of this PR, guess this could do with updating now we've got  rid of the original version

Reply:
Good point